### DISCUSSION TITLE
17/05/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#158] [Daily Sprint]
Que hice
Que hago
Impedimento

**Comment by Pamela198713** [comment]
Ayer estuve en clases de tecnología. Vi los videos explicativos sobre la web. 
Hoy los revisaré a la salida del cole. Está la revisión con la profe Viviana. 
Creo que nada, por ahí el tiempo

**Comment by rpgrca** [comment]
Ayer pude agregar el catálogo a la app, trae los datos del servidor. Hoy quiero estar en la revisión aunque todavía no solucionaron lo de la baja de inscripción. Me gustaría empezar con lo del carrito, que se puedan agregar y quitar cosas de ahí aunque eso requiere que funcione la parte de cookies que se usaba en Angular así que voy a investigar cómo se hace.

**Comment by fmdcba** [comment]
Hoty estuve trabajando en la activity de login según los diseños de figma:
![image](https://github.com/enbalde-ispc/enbalde-ispc/assets/95236196/47426a42-3d09-4d42-8503-103060ff4097)
Ahora estoy esperando para entrar a la review del grupo y luego planeo continuar con las pantallas
